docs/release-notes/4515-fix-test-submit.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fix**
+
+-  When creating a `test` experiment the container storage path was not being set correctly.
+   Also fixes the docstring which incorrectly claimed that only one slot would be used.
```

Comment:
nit: Since the docstring was not published in our docs, I would consider this detail about the docstring not important enough for release notes.

Definitely worth fixing though, just not for release notes.

Reply:
ok, done